docs/website/docs/dlt-ecosystem/verified-sources/airtable.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Airtable
+description: dlt verified source for Airtable
+keywords: [airtable api, airtable verified source, airtable]
+---
+
+# Airtable
+
+Airtable is a cloud-based platform that merges spreadsheet and database functionalities for easy
+data management and collaboration.
+
+This Airtable `dlt` verified source and
+[pipeline example](https://github.com/dlt-hub/verified-sources/blob/master/sources/airtable_pipeline.py)
+loads data using “Airtable API” to the destination of your choice.
+
+Sources and resources that can be loaded using this verified source are:
+
+| Name              | Description                                |
+| ----------------- | ------------------------------------------ |
+| airtable_source   | Retrieves tables from an airtable base     |
+| airtable_resource | Retrives data from a single airtable table |
+
+## Setup Guide
+
+### Grab Airtable personal access tokens
+
+1. Click your account icon top-right.
+1. Choose "Developer Hub" from the dropdown.
+1. Select "Personal access token" on the left, then "Create new token".
+1. Name it appropriately.
+1. Add read scopes for "data records" and "schema bases".
+1. Add a base to specify the desired base for data access.
+1. Hit "Create token" and save the displayed token securely for credential use.
+
+### Grab Airtable IDs
```

Comment:
Same as above 
```suggestion
### Obtain Airtable IDs
```

Reply:
Thanks but Grab is used in all docs. So cannot replace here!